setup.php

```diff
@@ -143,13 +143,24 @@ function cloneClicked() {
 <span class="text">
 
 <?php
+// Checks if the server's PHP version and the client's browser are compatible with OpenEMR:
```

Comment:
@bradymiller I only am using this in the one file for now as an example. I'm interested in your feedback before copy/pasting it elsewhere :)